WhatsNew.Cli/README.md

```diff
@@ -122,19 +122,31 @@ The following properties are supported in the JSON configuration file.
-| `minAdditionsToFile` | The minimum number of lines changed that a pull request file must contain before being included. Default value: `75` | `"minAdditionsToFile": 62` |
+| Property                    | Description | Example |
+| --------------------------- | ----------- | ------- |
+| `omitPullRequestTitles`     | A flag indicating whether to display pull request titles in the generated Markdown file. Default value: `false` | `"omitPullRequestTitles": true` |
+| `labels`                    | A list of GitHub label filters to apply. The label filters will be converted to a space-delimited string. Default value: `[]` | `"labels": [ "label:cognitive-services/svc" ]` |
+| `maxFilesChanged`           | The maximum number of changed files that a pull request can contain before being ignored. Default value: `75` | `"maxFilesChanged": 50` |
+| `minAdditionsToFile`        | The minimum number of lines changed that a pull request file must contain before being included. Default value: `75` | `"minAdditionsToFile": 62` |
 | `pullRequestTitlesToIgnore` | A comma-delimited list of regular expressions matching pull request titles to ignore. | `"pullRequestTitlesToIgnore": [ "^Confirm merge from repo_sync_working_branch", "^Repo sync for protected CLA branch" ]` |
 
 ### `docLinkSettings` properties
 
-| Property              | Description | Example |
-| --------------------- | ----------- | ------- |
-| `linkFormat`* | The Markdown format to use when creating links to docs. Possible values: `relative`, `siteRelative`, `xref`.<br><br>`siteRelative` links don't include file extensions; `relative` links do. | `"linkFormat": "relative"` |
+| Property             | Description | Example |
+| -------------------- | ----------- | ------- |
+| `linkFormat`*        | The Markdown format to use when creating links to docs. Possible values: `relative`, `siteRelative`, `xref`.<br><br>`siteRelative` links don't include file extensions; `relative` links do. | `"linkFormat": "relative"` |
 | `relativeLinkPrefix` | The path that prefixes the doc link. Required when `linkFormat` is set to `relative` or `siteRelative`. | `"relativeLinkPrefix": "/dotnet/"` |
 
 *Indicates a required property
+
+### `navigationOptions` properties
+
+These options apply in the 2.0 release of the What's New tool. The 2.0 release will update nodes in a TOC and an index file in addition to creating the What's New file. That requires config options for how many What's New files to keep publishing, and where to find the YML nodes to update.
+
+| Property                  | Description | Example |
+| ------------------------- | ----------- | ------- |
+| `maximumNumberOfArticles` | The maximum number of "What's new" articles or sections to keep. Once the value is reached, the older article or section gets deleted from your local repository storage. Default value: `3` | `"maximumNumberOfArticles": 6` |
```

Comment:
I found that I had to specify these (or it failed schema validation), and that I couldn't put `""` as the value. I had to put `" "`.  Is there a way to make them optional?

Reply:
I'll defer this one. I wrote #255 for general updates to the config object.